connector_sdk.proto

```diff
@@ -67,12 +67,12 @@ message TableSelection {
 message UpdateResponse {
     oneof operation {
         Record record = 1;
-        SchemaChange schema_change = 2;
+        ConnectorSchemaChange schema_change = 2;
```

Comment:
Please don't rename fields without talking to me first. Let's rename this back to what it was

Reply:
updated